src/reporters/consola.js

```diff
@@ -1,7 +1,7 @@
 'use strict'
 
 const { colors, clearConsole } = require('../utils/log')
-const consola = require('consola')
+const { consola } = require('consola')
```

Comment:
This is an unnecessary change btw. Having default export is more backward compatible.

Reply:
Yeah, but you (Nuxt team) recommends the switch: https://github.com/unjs/consola/releases/tag/v3.0.0

Also, Consola 3.x is in dependencies so it's guaranteed to be used by this code.

Reply:
I guess I am aware 😆 Yes, still a default export for old dependencies like this worth since older consola version can be hoisted. npm ecosystem is funny...

Reply:
I think hoisting would only potentially be an issue for a code running from webpack context. This should probably only run at build time so I would think should be guaranteed to run its own unhoisted dependency (assuming that it's different from the hoisted one).

Unless someone uses yarn `resolutions` then all bets are off but I think that's user's choice.

In any case, do you want me to change it?
Note that this is one of many dependencies with Consola so it's not likely that all will consistently follow the same way.

Reply:
Yes please changing to default export would be nice. And yes this is almost the only dependency I am asking to change considering its age. Hoisting can still happen within `node_modules`.